.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
   "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.quickSuggestions": {
+    "strings": true
+  },
```

Comment:
Enables VS Code autocomplete suggestions within string literals - particularly useful for Tailwind CSS class names in template strings and className props